README.md

```diff
@@ -27,8 +27,8 @@ up to you to manage.
    services:
      backend:
        ports:
-         - HOST_PORT:8080
+         - "8080:8080"
    ```
-5. `docker compose up -d`
+5. `docker-compose up -d`
```

Comment:
[`docker compose` is correct](https://docs.docker.com/engine/reference/commandline/compose/), if it's not available you need to update your docker

Reply:
I have the latest docker version but when I run without the dash it outputs the docker help docs

Reply:
https://docs.docker.com/compose/install/linux/

Reply:
I don't use the desktop version tho. I run my stuff on a remote server
When I also try to run `sudo apt-get install docker-compose-plugin` I get `E: Unable to locate package docker-compose-plugin`

Reply:
I guess it just doesn't work with my setup

Reply:
With Ubuntu server at least it works could be your package list is outdated or isn't on your version for me personally I use both v1 and v2 as I'm lasy asf and have the docker-compose up -d saved